onlyOffice/src/main/java/org/exoplatform/platform/qa/ui/onlyOffice/pageobject/OnlyOfficeEditingPage.java

```diff
@@ -30,13 +30,12 @@ public OnlyOfficeEditingPage(TestBase testBase) {
     /**
      * Check that document is opened on OnlyOffice Editor
      * @param documentName
-     * @param userName
+     * @param extension
      */
-    public void checkOpeningDocumentWithEditOnline(String documentName, String userName) {
-        $(byText(documentName + ".docx")).waitUntil(visible, openBrowserTimeoutMs);
+    public void checkOpeningDocumentWithEditOnline(String documentName,String extension) {
+        $(byText(documentName + extension)).waitUntil(visible, openBrowserTimeoutMs);
         $(".editor").waitUntil(visible, 60000).exists();
         switchTo().frame("frameEditor");
-        $("#title-user-name").waitUntil(exactText(userName), 120000);
-        $("#id_panel_left").waitUntil(visible, 120000);
+        $("#view-left-menu").parent().find(".tool-menu.left").waitUntil(visible, 120000);
```

Comment:
Why did you remove the check on title-user-name element ? It seems to be relevant to check that the user name is correctly displayed.

Reply:
I'm checking the opening, and the element I used before does not work with l pptx so I had to change a deeper element for the 3 file types, more than the user name

Reply:
ok but can we keep the check on the username as well ?

Reply:
I will check if it the same element of all types

Reply:
:+1: